app/views/annotation_categories/_annotation_text.html.erb

```diff
@@ -1,62 +1,73 @@
-<li id='annotation_text_<%= annotation_text.id %>'>
-  <div id='annotation_text_<%= annotation_text.id %>_control' style="word-break: break-all">
-    <%= markdown(annotation_text.content) %>
+<li id='annotation_text_<%= annotation_text['id'] %>'>
```

Comment:
The hash can be accessed with a string or a symbol. Which would be best? (this file is inconsistent in that regard)